perses/tests/test_repex.py

```diff
@@ -157,16 +155,15 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
                 # Set up repex simulation
                 reporter_file = os.path.join(temp_dir, f"{wt_name}-{mutant_name}.nc")
                 reporter = MultiStateReporter(reporter_file, checkpoint_interval=10)
-                hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinSplittingDynamicsMove(timestep=4.0 * unit.femtoseconds,
-                                                                                       collision_rate=1.0 / unit.picosecond,
-                                                                                       n_steps=50,
-                                                                                       reassign_velocities=False,
-                                                                                       n_restart_attempts=20,
-                                                                                       splitting="V R R R O R R R V",
-                                                                                       constraint_tolerance=1e-06),
-                                                                                       replica_mixing_scheme='swap-all',
-                                        hybrid_factory=htf,
-                                        online_analysis_interval=None)
+                hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=4.0 * unit.femtoseconds,
+                                                                              collision_rate=1.0 / unit.picosecond,
+                                                                              n_steps=50,
+                                                                              reassign_velocities=False,
+                                                                              n_restart_attempts=20,
+                                                                              constraint_tolerance=1e-06),
+                                         replica_mixing_scheme='swap-all',
+                                         hybrid_factory=htf,
+                                         online_analysis_interval=None)
                 hss.setup(n_states=36, temperature=300 * unit.kelvin, t_max=300 * unit.kelvin,
                           storage_file=reporter, minimisation_steps=0, endstates=True)
```

Comment:
I think we _may_ need to minimize a bit here, since things could get pathologically slow if there are NaNs.

Reply:
Resolved by https://github.com/choderalab/perses/pull/1065/files#r947881930